packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/CollectingLocality/index.js

```diff
@@ -90,8 +90,13 @@ class CollectingLocality extends PureComponent {
             />
           </Grid.Column>
         ) : (
-          <Grid.Column style={buttonStyle} width={6}>
-            <Button onClick={this.handleOpen} type="button">
+          <Grid.Column style={buttonStyle} width={5}>
+            <Button
+              floated="left"
+              onClick={this.handleOpen}
+              style={{ paddingLeft: '1.32em', paddingRight: '1.32em' }}
```

Comment:
there is also `buttonStyle` on the Grid.Column so I wonder if it is necessary to have style on both of them or if it is enough to do all styling on either the button or the column?